projects/plugins/jetpack/modules/masterbar/admin-menu/class-jetpack-admin-menu.php

```diff
@@ -55,6 +57,21 @@ public function get_preferred_view( $screen, $fallback_global_preference = true
 		return self::DEFAULT_VIEW;
 	}
 
+	/**
+	 * Get the link to the CPT screen. If SSO is enabled, the link will inside calypso,
+	 * otherwise it will be a link to wp-admin.
+	 *
+	 * @param string $post_type The ID of the post type.
+	 * @return string The link to Calypso if SSO is enabled or to WP Admin if SSO is disabled.
+	 */
+	public function get_cpt_menu_link( $post_type ) {
+		if ( \Jetpack::is_module_active( 'sso' ) ) {
+			return 'https://wordpress.com/types/' . $post_type . '/' . $this->domain;
```

Comment:
One thing worth noting: the post type will also need to `support = [ 'editor' ]` in order to be editable in Calypso.

<img width="834" alt="image" src="https://user-images.githubusercontent.com/36432/186270899-bff4859e-0307-48b4-a531-45d68eba8a27.png">

However, I tried removing support for `editor` and it elegantly redirected to the classic editor.

Although, do we need to accommodate `show_in_rest=true` too?

Reply:
> Although, do we need to accommodate show_in_rest=true too?

Good question! I guess Calypso is unable to edit CPTs if `show_in_rest=false` because it cannot interact with them through the REST API. In that case, let's ignore what I said in https://github.com/Automattic/jetpack/pull/25670#discussion_r953626247 and maybe link to WP-Admin as a fallback if `show_in_rest=false`?

Reply:
I've added the exception for `show_in_rest`, we will link to wp-admin on a new tab when `show_in_rest` is false.
<img width="253" alt="Screenshot 2022-08-24 at 13 11 17" src="https://user-images.githubusercontent.com/779993/186414957-04a70b96-502c-4756-a81e-bd56e310d11e.png">
t